### PR TITLE
Add New Quotes about Language

### PR DESCRIPTION
**Pull request by AdnanMuhib**
## ✅ New Quote Checklist
- [x] The quote(s) added are in alphabetical order and not ❌ duplicates.
- [x] The quote(s) added are about **learning a foreign language**.
- [x] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [x] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

## ✅ New Language Checklist
- [ ] There is ❌ no assigned issue or pull request for this new language.
- [ ] ➕ Added a new JSON object in the `server/data/languages.json` file.
- [ ] Added the **voice** 🎤 attribute supported by ResponsiveVoice API.
- [ ] Added the **flag** 🏳 attribute in the format of the twemoji-awesome CSS syntax.
- [ ] Added the **romanization** attribute for appropriate languages.
- [ ] ➕ Created the `{language.json}` file inside the `server/data/words` folder containing the list of the 3000 words.
- [ ] Removed duplicates and noted 🗒 them in the Pull Request.
- [ ] Increment `currentLanguagesCount` in `client/src/store/index.js`
- [ ] Add new language in `README.md`

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -23,6 +23,10 @@
     "author": "Amy Chua",
     "sentence": "Do you know what a foreign accent is? It’s a sign of bravery."
   },
+  {
+    "author": "Johann Wolfgang von Goethe",
```
Comment (Kvaibhav01) [inline server/data/quotes.json:27]: Please change the position of the quote alphabetically as suggested in the [contribution guide](https://github.com/jayehernandez/letra-extension/blob/master/docs/new_quote.md#-checklist).

File: server/data/quotes.json [hunk 2]
```diff
@@ -55,6 +59,10 @@
     "author": "Beatrice Taylor, Ojibwe",
     "sentence": "You’ve got to know your language to understand your culture."
   },
+  {
+    "author": "Czech proverb",
```
Comment (Kvaibhav01) [inline server/data/quotes.json:63]: This is a duplicate quote already added before. Remove it.

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Thanks for the PR @AdnanMuhib. Please update the requested changes.

File: server/data/quotes.json (hunk 1 above)
Comment (AdnanMuhib) [reply server/data/quotes.json:27]: @Kvaibhav01 there is ambiguity about the alphabetic order, If I take a look at the previously added quotes, I don't find them ordered.
Are they ordered based on the Starting Letter of the sentence or the First Letter of the Author Name?

Comment (Kvaibhav01) [reply server/data/quotes.json:27]: Hey, they're ordered on the first letter of the author's name.

Comment (AdnanMuhib) [reply server/data/quotes.json]: Thanks for Clarifying, a quick suggestion, it should be updated in the Contribution Guide as well as in the PR template.

File: server/data/quotes.json [hunk 3]
```diff
@@ -275,6 +275,10 @@
     "author": "Jhumpa Lahiri",
     "sentence": "A foreign language is like a frail, delicate muscle. If you don't use it, it weakens."
   },
+  {
+    "author": "Johann Wolfgang von Goethe",
+    "sentence": "Those who know nothing of foreign languages know nothing of their own."
```
Comment (jayehernandez) [inline server/data/quotes.json:280]: This is a duplicate quote (see line 295), please add a different quote.

**Review by jayehernandez — CHANGES_REQUESTED** [review]

File: server/data/quotes.json (hunk 3 above)
Comment (AdnanMuhib) [reply server/data/quotes.json]: Thanks for pointing, I was using ctrl+f against the whole sentence, didn't know that the same quote may exist rephrased.

Comment (jayehernandez) [reply server/data/quotes.json]: Thanks for changing! :)

**Review by jayehernandez — CHANGES_REQUESTED** [review]
Last comment! Please add it in alphabetical order by author's first name :)

**Comment by jayehernandez** [comment]
Closing stale PR

**Pull request closed without merging**